iOS14UIKitTour/iOS14UIKitTour/ViewControllers/SplitViewController.swift

```diff
@@ -1,7 +1,38 @@
 
 import UIKit
+import Combine
+import SwiftUI
 
 final class SplitViewController: UISplitViewController {
-
+    @ObservedObject var modelController: ModelController
+    
+    private var cancellables = Set<AnyCancellable>()
+    
+    init() {
+        modelController = ModelController()
+        
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
     
+    override func viewDidLoad() {
+    //        GithubAPI.fetch(.repositories)
+    //            .sink(receiveCompletion: { completion in
+    //                print(completion)
+    //            }, receiveValue: { (repositories: [Repository]) in
+    //
+    //
+    //            }).store(in: &cancellables)
+        
+    //        GithubAPI.fetch(.contributors("backtobasicsuikit"))
+    //            .sink(receiveCompletion: { completion in
+    //                print(completion)
+    //            }, receiveValue: { (contributors: [Contributor]) in
+    //                print(contributors)
+    //
+    //            }).store(in: &cancellables)
+    }
```

Comment:
This I get why you left it for me. 👍🏼
Which property do I use from a `Repository` type to fetch `.contributors(repo.XXX)`